addons/cookoff/functions/fnc_cookOff.sqf

```diff
@@ -113,22 +113,35 @@ if (local _vehicle) then {
 
         [_vehicle, _fnc_FlameEffect, 12] call _fnc_FlameEffect; // recursive function
 
+        private _randomPosition = [getPos _vehicle, 100, random 360] call BIS_fnc_relPos;
```

Comment:
`private _randomPosition = _vehicle getPos [100, random 360]`;

- is 100 meters too much?